content/doc/book/blueocean/activity.adoc

```diff
@@ -19,93 +19,72 @@ endif::[]
+
+This view does not allow runs to be edited or marked as favorites.
+To perform these actions, select the "<<branches, Branches>>" tab.
+
+=== Branches
+
+The Branches tab shows a list of all branches that have a completed or in-progress run in the current Pipeline.
+Each line in the list corresponds to a branch in source control, showing <<dashboard#pipeline-health, overall health of the branch>> based on the status of the most recent run, branch name, commit information, and when the run completed.
+
+image:blueocean/activity/branches.png[Branches tab of Activity view, role=center]
+
+Selecting a branch will bring up the <<pipeline-run-details#, Pipeline run details>> for the latest completed or in-progress run of that branch.
+
+. Pipelines where the latest run has been completed can be run again by selecting the *run* icon, represented by a icon:play[] in a circle.
++
+* In-progress runs can be aborted, and display a *stop* icon, which is represented by a &#9726; within a circle.
+. Selecting the *history* icon icon:history[] will allow you to view the run history for that branch.
+. The *edit* icon, represented by a icon:pencil[], opens the <<pipeline-editor#, pipeline editor>> for that branch.
+. The *favorite* &#9734; icon adds a branch to your favorites list on the <<dashboard#favorites, dashboard>>.
+On the dashboard a branch listed under favorites will show a solid star &#9733;.
+Deselecting it removes this branch from the favorites list.
+
+=== Pull Requests
+
+The Pull Requests tab shows a list of all pull requests for the current Pipeline, that have a completed or in-progress run.
+Each line in the list corresponds to a pull request in source control, which displays the status of the most recent run, run id, summary, author, and when the run completed.
 
 image:blueocean/activity/pull-requests.png[Activity Pull Requests view, role=center]
 
-Blue Ocean displays pull requests separately from branches,
-but otherwise, the Pull Requests list behaves similar to the Branches list.
-Clicking on a pull request in this list will bring up the
-<<pipeline-run-details#, Pipeline Run Details>>
-for the latest completed or in-progress Run of that pull request.
-"In Progress" Runs can be aborted from this list by clicking on the "Stop" symbol
-(a square "&#9724;" inside a circle).
-Pull requests whose latest Run has been completed can be run again by clicking the
-"Play" symbol (a triangle "&#9654;" inside a circle).
-Pull request do not display <<dashboard#pipeline-health, "Heath Icons">>
-and cannot be edited or marked as favorites.
-
-NOTE: By default, when a Pull Request is closed,
-Jenkins will remove the Pipeline from Jenkins (to be cleaned up at a later date),
-and Runs for that Pull Request will no longer be accessible from Jenkins.
-That can be changed by changing the configuration of the underlying Multi-branch Pipeline job.
+Blue Ocean displays pull requests and branches in separate tabs, but the lists themselves behave similarly.
```

Comment:
Is it a "but" or an "and"? Not sure if the statement is a contrast to the previous or not - up to you.
```suggestion
Blue Ocean displays pull requests and branches in separate tabs, and the lists behave similarly.
```

Reply:
I updated it to this:

```suggestion
Blue Ocean displays pull requests and branches separately, but the lists behave similarly.
```
Does that make a bit more sense or would it be better to have the and in this case? I believe the intent is to explain that while they are displayed in two places, they provide some of the same functionalities.